### DISCUSSION TITLE
Specify internal IP range for P2P VPN

### DISCUSSION THREAD

**Discussion by antongisli** [kairos-io/kairos#1893] [Q&A]
**Is your feature request related to a problem? Please describe.**
It isn't clear how the internal IP range is decided for a p2p network. For example, how to control how large the subnet is.

**Describe the solution you'd like**
When using p2p, we would like to know if it is possible to specify the internal IP range (used for the overlay/VPN), e.g. as a CIDR.

**Comment by jimmykarily** [comment]
It was discussed in a call, this is not possible right now. If it's needed please open a feature request ticket. cc @mudler
